### PR TITLE
[4.0] Remove references to JHtmlSidebar 

### PR DESCRIPTION
**Pull request by travisrisner**
Pull Request for Issue #24859

### Summary of Changes
Remove existing references to JHtmlSidebar, as noted on issue #24859

### CODE REVIEW

File: administrator/components/com_messages/Helper/MessagesHelper.php [hunk 1]
```diff
@@ -21,29 +21,6 @@
...
-			'index.php?option=com_messages&view=message&layout=edit',
-			$vName == 'message'
-		);
-
-		\JHtmlSidebar::addEntry(
-			Text::_('COM_MESSAGES_READ'),
-			'index.php?option=com_messages',
-			$vName == 'messages'
-		);
-	}
```
Comment (Quy) [inline administrator/components/com_messages/Helper/MessagesHelper.php]:
```suggestion
```

**Comment by Quy** [comment]
Here are some duplicate PRs #24867, #24866. Search `sidebar` for additional PRs.

**Comment by travisrisner** [comment]
> Here are some duplicate PRs #24867, #24866. Search `sidebar` for additional PRs.

Would I be better off closing this and handling the ones that don't have an existing PR?

**Comment by Quy** [comment]
Yes close and open PRs for ones that are not done yet. 
Also, you can test existing PRs since each PR needs 2 successful tests.
Thanks.

**Comment by Quy** [comment]
Thank you for testing! Here are 2 more to test if you have the time: #24870 #24868
Related PRs #24020 #23366

**Pull request closed without merging**